packages/core/src/rum/__tests__/DdRum.test.ts

```diff
@@ -72,10 +77,10 @@ describe('DdRum', () => {
         });
 
         test('calls the native SDK when called with old API with default values', async () => {
-            await DdRum.startAction('scroll', 'page_old_api');
+            await DdRum.startAction(RumActionType.SCROLL, 'page_old_api');
             await DdRum.stopAction();
             expect(NativeModules.DdRum.stopAction).toHaveBeenCalledWith(
-                'scroll',
+                RumActionType.SCROLL,
```

Comment:
Should we do these replacements as part of this PR or separate it into its own PR?

Reply:
I have thought about it, but I have decided to push it with this PR since these are pretty simple changes and they are in the scope of the newly implemented tests (kind of). Also, it's easy to verify that the tests still execute and pass correctly, so in my opinion we can keep it in this PR.